slack-api-client/src/main/java/com/slack/api/methods/impl/TeamIdCache.java

```diff
@@ -18,6 +20,14 @@
 @Slf4j
 public class TeamIdCache {
 
+    // As the tokens issued for "Sign in with Slack" does not work with auth.test API method,
+    // we skip creating team_id cache for these API methods.
+    public static final List<String> METHOD_NAMES_TO_SKIP_TEAM_ID_CACHE_RESOLUTION = Arrays.asList(
+            "auth.revoke",
```

Comment:
@seratch regarding your comment in this PR:

>  For auth.revoke, as the token will be disabled by this call, you don't need to worry about rate limited errors with it any more. 

It looks like the auth.revoke API accepts a `test` parameter, like a kind of "dry run" execution that wouldn't actually revoke the token. I am not sure if you considered that, but does that change the consideration around removing tracking call frequency for this API?

Acceptable answers include "no, Fil, go away" 🤪

Reply:
@filmaj haha, of course, I don't say "go away" 😸 

You've pointed out a great point! Indeed, the ignorance here may allow rate limited errors. However, if a developer repeatedly performs auth.revoke for a specific workspace (as this endpoint is [tier 3](https://api.slack.com/docs/rate-limits#tier_t3), specifically 50+ call per minute for a workspace), there should be many other issues in addition to the rate limited error due to the calls. Perhaps, the developer will need to improve the operation to make it more meaningful in the case.

The only realistic use case that I can think of would be a batch processing job that cleans up the tokens in a database and performs a lot of auth.revoke as necessary. 

Indeed, the code with this change cannot handle the situation but I don't think that it's so common. Also, unfortunately, we don't have any means to distinguish whether a token is for normal OAuth or for "Sing in with Slack" without performing auth.test or whatever. For this reason, I think that simply skipping auth.revoke here should be reasonable enough. If the Slack platform provides more light-weight way to verify if a token is for SIWS or not, we can improve this part then.

Reply:
It certainly is an extreme edge case... completely agree it wouldn't be common. I will now go away 😂  Nice work with the PR, thank you for writing this up!